rtengine/iplocallab.cc

```diff
@@ -1648,12 +1648,13 @@ static void calcLocalParams(int sp, int oW, int oH, const LocallabParams& locall
     lp.hlcompthr = locallab.spots.at(sp).hlcomprthresh;
     lp.expcomp = LIM(locallab.spots.at(sp).expcomp, -2.0, 4.0); //to prevent crash with Old pp3 with integer
     //increase sensitivity for low values
-    float proexp = lp.expcomp;
-    if (std::fabs(proexp) < 0.6f) {
-        float interm = std::fabs(proexp) / 0.6f;
+/*    float proexp = lp.expcomp;
+    if (proexp > 0.f && proexp < 0.6f) {
+        float interm = proexp / 0.6f;
         interm = pow(interm, 3.f);
         lp.expcomp = proexp * interm;
     }
+*/    
```

Comment:
Hi @Desmis if this is no longer needed, it can be removed instead of commented. If necessary the git history can restore the code.

Reply:
OK, I remove...

Jacques